tools/codegen/pkg/interfacegen/template/interface.go

```diff
@@ -29,40 +29,46 @@ import (
 // Fully qualified name of this template
 const TemplateName = "{{.PackageName}}.{{.Name}}"
 
-// Instance is constructed by Mixer for '{{.PackageName}}.{{.Name}}' template.{{if ne .TemplateMessage.Comment ""}}
+// Instance is constructed by Mixer for the '{{.PackageName}}.{{.Name}}' template.{{if ne .TemplateMessage.Comment ""}}
 //
 {{.TemplateMessage.Comment}}{{end}}
 type Instance struct {
+  // Name of the instance as specified in configuration.
   Name string
   {{range .TemplateMessage.Fields}}
   {{.Comment}}
   {{.GoName}} {{replaceGoValueTypeToInterface .GoType}}
   {{end}}
 }
 
-// {{.Name}}HandlerBuilder must be implemented by adapter code if it wants to
-// process data associated with the template.
+// {{.Name}}HandlerBuilder must be implemented by adapters if they want to
```

Comment:
nit: remove the "if they want", just "FooHandlerBuilder must be implemented by adapters to process data associated with the Foo template." reads better IMO.